hack/kiali/func-sm.sh

```
@@ -0,0 +1,442 @@
+  echo -n "Waiting."
+  while ! ${OC} get deployment -n ${OLM_OPERATORS_NAMESPACE} -o name | grep -E 'sail|servicemesh|istio' >& /dev/null ; do echo -n '.'; sleep 1; done
+  echo "done."
+  local servicemesh_deployment="$(${OC} get deployment -n ${OLM_OPERATORS_NAMESPACE} -o name | grep -E 'sail|servicemesh|istio')"
+
+  infomsg "Waiting for operator deployments to start..."
+  for op in ${servicemesh_deployment}
+  do
+    infomsg "Expecting [${op}] to be ready"
+    echo -n "Waiting."
+    local readyReplicas="0"
+    while [ "$?" != "0" -o "$readyReplicas" == "0" ]
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

(SC2166)